src/components/Camera/style.scss

```diff
@@ -117,8 +119,20 @@
   }
 
   .docLiveCaptureFrame & {
-    height: auto;
-    margin: 0;
+    left: 0;
+    top: 50%;
+    margin: auto;
+    transform: translate(0%, -50%);
+
+    @media (orientation: landscape) {
+      height: auto;
+      width: 100%;
+    }
+
+    @media (orientation: portrait) {
+      height: auto;
+      width: 100%;
+    }
```

Comment:
Just realised this is duplicated & unnecessarily in separate media queries 🙈

Reply:
Fixed